kyuubi-main/src/main/scala/org/apache/kyuubi/operation/ExecuteStatement.scala

```diff
@@ -91,8 +91,10 @@ class ExecuteStatement(
       isComplete = true
       remoteState match {
         case INITIALIZED_STATE | PENDING_STATE | RUNNING_STATE =>
-          isComplete = false
-          statusResp = client.GetOperationStatus(statusReq)
+          try {
```

Comment:
shall we wrap the whole method？ there are still other places calling the `client`